[2024-08-01T17:09:36.194+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-08-01T17:09:36.234+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_dag.create_dim_company manual__2024-08-01T17:09:27.168537+00:00 [queued]>
[2024-08-01T17:09:36.249+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_dag.create_dim_company manual__2024-08-01T17:09:27.168537+00:00 [queued]>
[2024-08-01T17:09:36.250+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 6
[2024-08-01T17:09:36.268+0000] {taskinstance.py:2330} INFO - Executing <Task(PostgresOperator): create_dim_company> on 2024-08-01 17:09:27.168537+00:00
[2024-08-01T17:09:36.296+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:62: DeprecationWarning: This process (pid=777) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-08-01T17:09:36.301+0000] {standard_task_runner.py:64} INFO - Started process 785 to run task
[2024-08-01T17:09:36.315+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'etl_dag', 'create_dim_company', 'manual__2024-08-01T17:09:27.168537+00:00', '--job-id', '133', '--raw', '--subdir', 'DAGS_FOLDER/etl_dag.py', '--cfg-path', '/tmp/tmp783vfyl1']
[2024-08-01T17:09:36.341+0000] {standard_task_runner.py:91} INFO - Job 133: Subtask create_dim_company
[2024-08-01T17:09:36.485+0000] {task_command.py:426} INFO - Running <TaskInstance: etl_dag.create_dim_company manual__2024-08-01T17:09:27.168537+00:00 [running]> on host e27a9ccca098
[2024-08-01T17:09:36.810+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Gabriel Almeida' AIRFLOW_CTX_DAG_ID='etl_dag' AIRFLOW_CTX_TASK_ID='create_dim_company' AIRFLOW_CTX_EXECUTION_DATE='2024-08-01T17:09:27.168537+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-08-01T17:09:27.168537+00:00'
[2024-08-01T17:09:36.853+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-08-01T17:09:36.870+0000] {sql.py:282} INFO - Executing: 
            CREATE TABLE IF NOT EXISTS DimCompany (
                id_company INT PRIMARY KEY,
                company VARCHAR(255)
            );
        
[2024-08-01T17:09:36.949+0000] {base.py:84} INFO - Using connection ID 'postgres_conn' for task execution.
[2024-08-01T17:09:37.030+0000] {base.py:84} INFO - Using connection ID 'postgres_conn' for task execution.
[2024-08-01T17:09:37.058+0000] {sql.py:495} INFO - Running statement: 
            CREATE TABLE IF NOT EXISTS DimCompany (
                id_company INT PRIMARY KEY,
                company VARCHAR(255)
            );
        , parameters: None
[2024-08-01T17:09:37.080+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-08-01T17:09:37.103+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=etl_dag, task_id=create_dim_company, run_id=manual__2024-08-01T17:09:27.168537+00:00, execution_date=20240801T170927, start_date=20240801T170936, end_date=20240801T170937
[2024-08-01T17:09:37.194+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2024-08-01T17:09:37.250+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
